NuGet.Services.Dashboard.Operations/Tasks/DashBoardTasks/AzureManagementTasks/CreateTrafficManagerStatusOverviewReportTask.cs

```diff
@@ -29,36 +29,48 @@ public override void ExecuteCommand()
             request.Headers.Add("x-ms-version: 2014-02-01");
             request.PreAuthenticate = true;
             request.Method = "GET";
-            WebResponse respose = request.GetResponse();
+            WebResponse response = request.GetResponse();
             //Schema of the response would be as specified in http://msdn.microsoft.com/en-us/library/azure/hh758251.aspx
-            Console.WriteLine(respose);
```

Comment:
It was replaced by the log of the actual contents below. This line did nothing but log some object type name.